packages/alfa-cli/bin/alfa/profiler.ts

```diff
@@ -0,0 +1,59 @@
+import * as inspector from "inspector";
```

Comment:
Is the inspector file missing from the PR?

Reply:
It's a Node.js built-in module: https://nodejs.org/api/inspector.html

Reply:
🙈